src/lang/en.json

```diff
@@ -21,6 +21,13 @@
   "success-section-instruction-2": "One form to let the content administrator edit it",
   "success-section-instruction-3": "One view so it can display as you want it for the site visitors",
   "delete-section-type": "Are you sure you want to delete this section type: ",
+  "authorize-section-type": "Authorize this section type: ",
+  "un-authorize-section-type": "You will block the configurable section, are you sure you want to continue ?",
   "Confirm": "Confirm",
-  "Cancel": "Cancel"
+  "Cancel": "Cancel",
+  "pkey": "Private key",
```

Comment:
these fields are coming from the third party itself and not from Sections engine, it should not be part of the library